src/App.js

```diff
@@ -1,14 +1,100 @@
 import React from 'react';
 
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
+
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+
+    this.state = {
+      todos: [],
+      newTodo: "",
+      filter: ""
+    }
+  }
+
+  //Update new todo
+  updateNewTodo = event => {
+    this.setState({
+      newTodo: event.target.value
```

Comment:
remember to make this dynamic! its just good practice even if you only have one input.
```suggestion
      [event.target.name]: event.target.value
```